thingif/src/main/java/com/kii/thingif/ThingIFAPI.java

```diff
@@ -196,6 +196,29 @@ public Builder setInstallationID(
             return this;
         }
 
+        /**
+         * Instantiate new Builder without Context.
+         * This method is for internal use only. Do not call it from your application.
+         *
+         * @param app Kii Cloud Application.
+         * @param owner Specify who uses the ThingIFAPI.
+         * @return Builder instance.
+         */
+        @NonNull
+        public static Builder _newBuilder(
```

Comment:
I think that the reason to create this method is keys of `actionTypes` map is `String` concatenated action name and alias name. What about creating new class representing action name and alias name such as following:

```java

public class AliasActionKey {
    @NonNull String actionName;
    @NonNull String aliasName;
}
```

Using this class enables to use `ThingIFAPI.Builder.registerAction()`. As a result, We remove `_newBuilder()` method.

After learning swift, it is undeniable for me to create public methods which is only used in internal but if we do not have to create such methods, it is more better way, I think.

This is my taste. You can ignore this suggestion:)